### PR TITLE
create admin_word pages

### PR DESCRIPTION
**Pull request by YutakaYamasaki**

### CODE REVIEW

**Comment by YutakaYamasaki** [comment]
Sorry . 
I was unable to unify the indentation of words_controller so I committed it again.

File: app/views/admin/categories/_category.html.erb [hunk 1]
```diff
@@ -1,6 +1,6 @@
     <tr>
         <td><%= link_to category.title, admin_category_path(category) %></td>
         <td><%= link_to "EDIT", edit_admin_category_path(category) %></td>
-        <td><%= link_to "Add Word", "" %></td>
+        <td><%= link_to "Add Word", new_admin_category_word_path(category) %></td>
```
Comment (inahsanjuan) [inline app/views/admin/categories/_category.html.erb]: @YutakaYamasaki Try add a classes to your linkts to make it more like a button, something like:
`btn btn-primary btn-sm`

File: app/views/admin/categories/_category.html.erb [hunk 2]
```diff
@@ -1,6 +1,6 @@
     <tr>
```
Comment (inahsanjuan) [inline app/views/admin/categories/_category.html.erb:1]: You can format your table like this:
image

File: app/views/admin/words/_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        </div>
+        <div class= "form-group">
+                <%= f.label :Word_answer %>
+                <%= f.fields_for :word_answers do |i| %>
+                    <%= i.check_box :correct %>
+                    <%= i.text_field :content,  class: "form-control" %>
+                <% end %>
+        </div>
+
+                <%= f.submit "Create new word", class: "btn btn-primary form-control" %>
```
Comment (inahsanjuan) [inline app/views/admin/words/_form.html.erb]: @YutakaYamasaki Fix indentation in this area.

File: app/controllers/admin/words_controller.rb [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    @category = Category.find(params[:category_id])
+    @word = @category.words.build
+    3.times{@word.word_answers.build}
+  end
+
+  def create
+    @category = Category.find(params[:category_id])
+    @word = @category.words.build(word_params)
+    if @word.save
+      flash[:succees] = "Create Word."
```
Comment (inahsanjuan) [inline app/controllers/admin/words_controller.rb]: @YutakaYamasaki Change `succees` to `success`.

File: app/views/admin/words/_word.html.erb [hunk 5]
```diff
@@ -0,0 +1,6 @@
+    <tr>
+        <td><%= word.content %></td>
+        <td><%= word.correct_answer.content %></td>
+        <td><%= link_to "edit", edit_admin_category_word_path(@categories, word) %></td>
+        <td><%= link_to "delete", admin_category_word_path(@categories, word), method: "delete" %></td>
```
Comment (inahsanjuan) [inline app/views/admin/words/_word.html.erb]: @YutakaYamasaki Try to change format of display, something like this: 
image

File: app/views/admin/words/edit.html.erb [hunk 6]
```diff
@@ -0,0 +1,8 @@
+<%=  provide(:title,"Update Word") %>
+
+<div class= "row">
+    <div class="col-center">
+        <div class="text-center"><%= yield(:title) %></div>
```
Comment (inahsanjuan) [inline app/views/admin/words/edit.html.erb]: @YutakaYamasaki Update this element to `<h1>`

**Review by inahsanjuan — CHANGES_REQUESTED** [review]
- There is no link to go to the `words#index`

**Pull request merged**